Makefile

```diff
@@ -32,6 +32,7 @@ build:
 	$(GOENV) $(GOBUILD) -o ./out/lifecycle/exporter -a ./cmd/exporter
 	$(GOENV) $(GOBUILD) -o ./out/lifecycle/cacher -a ./cmd/cacher
 	$(GOENV) $(GOBUILD) -o ./out/lifecycle/launcher -a ./cmd/launcher
+	$(GOENV) $(GOBUILD) -o ./out/lifecycle/rebaser -a ./cmd/rebaser
```

Comment:
We probably want to document this new artifact in our README.md

Reply:
ohh..good point. On it.